server/adapters/prismic.js

```diff
@@ -324,91 +338,107 @@ const getDataForApi = function ( req, listener ) {
+                        if (!data.item) {
+                            reject(`The document with UID "${isNaviNoForm ? navi.uid : uid}" could not be found by ${core.config.skylab.name}.`);
                         }
                     }
 
-                    resolve( data );
+                    resolve(data);
                 }
             };
-            const fail = function ( error ) {
-                reject( error );
+            const fail = function (error) {
+                reject(error);
             };
 
+            let newType = type;
+
+            if (type === "home") {
+                newType = "project";
+            } else if (type === "studio") {
+                newType = "page";
+            } else if (type === "work") {
+                newType = "project";
+            } else if (type === "play") {
+                newType = "blog";
+            }
+
             // query: type?
-            if ( isNaviNoForm ) {
-                query.push( prismic.Predicates.at( "document.type", navi.type ) );
-                query.push( prismic.Predicates.at( "document.id", navi.id ) );
+            if (isNaviNoForm) {
+                query.push(prismic.Predicates.at("document.type", navi.type));
+                query.push(prismic.Predicates.at("document.id", navi.id));
 
-            } else if ( !cache.api.data.forms[ type ] ) {
+            } else if (!cache.api.data.forms[type]) {
                 // Only if type? is NOT a search form collection
-                query.push( prismic.Predicates.at( "document.type", type ) );
+                query.push(prismic.Predicates.at("document.type", newType));
             }
 
             // query: pubsub?
-            if ( listener && listener.handlers.query ) {
-                query = listener.handlers.query( prismic, cache.api, query, cache, req );
+            if (listener && listener.handlers.query) {
+                query = listener.handlers.query(prismic, cache.api, query, cache, req);
             }
 
             // query: promise?
-            if ( query instanceof Promise ) {
-                query.then( done ).catch( fail );
+            if (query instanceof Promise) {
+                query.then(done).catch(fail);
 
             } else {
                 // query?
-                if ( query.length ) {
-                    form.query( query );
-                }
+                // if (query.length) {
+                //     form.query(query);
+                // }
 
                 // ordering: pubsub?
-                if ( listener && listener.handlers.orderings ) {
-                    listener.handlers.orderings( prismic, cache.api, form, cache, req );
-                }
+                // if (listener && listener.handlers.orderings) {
+                //     listener.handlers.orderings(prismic, cache.api, form, cache, req);
```

Comment:
We can't use this orderings anymore as far as I can tell so I'm not sure if any of the content listings like work, play or the homepage are no longer in the desired order. An alternative for the short term would be to do a sort here on the results array against either `firstPublicationDate` or `lastPublicationDate` maybe?